lib/services/collections_service.dart

```diff
@@ -90,12 +91,14 @@ class CollectionsService {
   // within the collection.
   Future<List<Collection>> sync() async {
     _logger.info("Syncing collections");
+    final EnteWatch watch = EnteWatch("syncCollection")..start();
```

Comment:
Should we remove the watch on production?

Reply:
Yes, I will will gradually remove them and by default turn off the logs.